src/gui/MainWindow.cpp

```diff
@@ -245,6 +245,9 @@ MainWindow::MainWindow() :
 
 	connect(Engine::getSong(), SIGNAL(modified()), SLOT(onSongModified()));
 	connect(Engine::getSong(), SIGNAL(projectFileNameChanged()), SLOT(onProjectFileNameChanged()));
+
+	maximised = ( isMaximized() ? true : false );
```

Comment:
This line will be replaced with just `maximised = isMaximised()`, I don't know what got over me here xD

Reply:
Fixed this up.